tests/CppUTestExt/IEEE754PluginTest.cpp

```diff
@@ -27,59 +27,67 @@
 
 /* The IEEE754ExceptionFlags plugin by definition requires C++11 */
 
-#ifdef CPPUTEST_COMPILER_FULLY_SUPPORTS_CXX11
-
 #include "CppUTest/CommandLineTestRunner.h"
 #include "CppUTest/TestHarness.h"
 #include "CppUTest/TestRegistry.h"
 #include "CppUTest/TestTestingFixture.h"
 #include "CppUTestExt/IEEE754ExceptionsPlugin.h"
-#include <cfenv>
 
 extern "C" { 
     #include "IEEE754PluginTest_c.h"
 }
 
+#if CPPUTEST_USE_STD_C_LIB
```

Comment:
Why not just putting all tests in the #if. Thats how we do this in other places. No need to ignore tests for functionality that doesn't exists :)